### PR TITLE
Migrate bug report issue template to GitHub form

### PR DESCRIPTION
**Pull request by Maxr1998**
You can try out the new form [here](https://github.com/Maxr1998/jellyfin-android/issues/new?assignees=&labels=bug&template=bug-report.yaml).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      description: Manufacturer, model
+      placeholder: Google Pixel 5, Samsung Galaxy S21
+    validations:
+      required: true
+  - type: input
+    id: android-version
+    attributes:
+      label: Android version
+      description: Version of the OS and other information (e.g. custom ROM / OEM skin)
+      placeholder: Android 11, Android 10
```
Comment (nielsvanvelzen) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:69]: Maybe extend the placeholder to show an example for a custom ROM like this;
```suggestion
      placeholder: Android 11, LineageOS 10
```

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    validations:
+      required: true
+  - type: input
+    id: server-version
+    attributes:
+      label: Jellyfin server version
+      placeholder: 10.7.6
```
Comment (nielsvanvelzen) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:76]: "if unstable please specify the commit hash"

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      label: Logs
+      description: |
+        Please paste your client logs (logcat) here, *NOT* server logs (in most cases).
+        Learn how to capture those logcats [here](https://wiki.lineageos.org/logcat.html).
+      placeholder: Paste logs…
+      render: shell
+  - type: input
+    id: app-version
+    attributes:
+      label: Applicaiton version
```
Comment (nielsvanvelzen) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:33]:
```suggestion
      label: Application version
```

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        Instead, I expect …
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: |
+        Please paste your client logs (logcat) here, *NOT* server logs (in most cases).
+        Learn how to capture those logcats [here](https://wiki.lineageos.org/logcat.html).
```
Comment (nielsvanvelzen) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:27]: hide your server url, access token, passwords etc

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+      required: true
+  - type: checkboxes
+    id: video-player
+    attributes:
+      label: Which video player implementations does this bug apply to?
+      description: |
+        *If applicable.* Video player can be switched in the client settings of the app.
+      options:
+        - label: Web player (default)
+        - label: Integrated player (ExoPlayer)
```
Comment (nielsvanvelzen) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:87]: Called "Native player" in the app

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+        - label: External player (mpv, VLC)
```
Comment (nielsvanvelzen) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:88]: The app uses MX Player and VLC as examples

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+      description: Choose the appropriate app store or installtion method.
+      options:
+        - Google Play
+        - F-Droid
+        - Amazon Appstore
+        - Sideloaded APK (GitHub, etc.)
+  - type: dropdown
+    id: app-variant
+    attributes:
+      label: Which variant are you using?
```
Comment (nielsvanvelzen) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:51]: Not sure if we need to ask for the variant if we're already asking for the install method. If we add two install methods (sideload libre, sideload proprietary) we should have enough info

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+      label: Applicaiton version
+      description: The version of the installed Jellyfin Android app.
+      placeholder: 2.3.0
+    validations:
+      required: true
+  - type: dropdown
+    id: installation-source
+    attributes:
+      label: Where did you install the app from?
+      description: Choose the appropriate app store or installtion method.
```
Comment (nielsvanvelzen) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:42]:
```suggestion
      description: Choose the appropriate app store or installation method.
```

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 5 above)
Comment (Maxr1998) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:87]: I thought we changed this. Weird.

Comment (Maxr1998) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:87]: Oh, only in the blog post. We should change it in the app too. I'd still merge this as is.

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 6 above)
Comment (Maxr1998) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:88]: I could just add all three ig.

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 7 above)
Comment (Maxr1998) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:51]: Good idea.

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 8 above)
Comment (Maxr1998) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:42]: The amount of typos is way too high. Not sure why Android Studio didn't detect them.

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 5 above)
Comment (nielsvanvelzen) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:82]: Don't forget to rename it in-app then :p

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 8 above)
Comment (nielsvanvelzen) [reply .github/ISSUE_TEMPLATE/bug-report.yaml]: Maybe you disabled the spellchecker? 🤔

Comment (Maxr1998) [reply .github/ISSUE_TEMPLATE/bug-report.yaml]: I don't think so? But maybe it didn't apply to that yaml file.

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 5 above)
Comment (Maxr1998) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:82]: Will do that in another PR.

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**